src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  // create array with only men or if century identificated take
+  // only men borned in this century
+  const menList = people.filter(person => century
+    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+    : person.sex === 'm');
+  // quantity of choosed men
+  const menQuantity = menList.length;
+  // with reduce calculate man ages
+  const menAvarageAge = menList.reduce((sumOfAges, man) => (
+    sumOfAges + (man.died - man.born)
+  ), 0) / menQuantity;
+
+  return +menAvarageAge.toFixed(2);
```

Comment:
you repeat the same logic three times in your code, let's create a function for these calculations